packages/block-library/src/paragraph/editor.scss

```diff
@@ -17,3 +17,23 @@
 		}
 	}
 }
+
+.components-popover.wp-block-paragraph__drop-zone {
+	.components-popover__content {
+		border: none;
+		outline: none;
+		box-shadow: none;
+	}
```

Comment:
This is also what `BlockPopover` does, so it's probably worth making this a proper style option of the `Popover` component in a separate PR.

Popover does already have an `isAlternate` class, but I know `Button` was refactored to a `variation` prop, so it could be worth doing something like that.

I guess the other option is that there's a lower level `Floating` component that has no visible UI, and then `Popover` is the component that's presents a UI and uses `Floating`.

Marco (@ciampo) will probably have some thoughts on this.

Reply:
> we could look into adding a `variant` props to the `Popover` component (which also replaces the `isAlternate` prop), or consider alternatives (like exposing a lower level unstyled floating element).

Agreed 👍

Reply:
We have this pattern (overriding `.components-popover__content`) all over the codebase already 😅. Let's keep this as-is and work on a different solution as a follow-up to the Popover component improvement. WDYT?

Reply:
> We have this pattern (overriding .components-popover__content) all over the codebase already 

I know, unfortunately it has been a pattern for a long time and in the past months we've been  trying to educate folks to avoid it as much as possible 😓 

Let's stick with your proposed plan for this PR